classes/app.rb

```diff
@@ -65,6 +72,14 @@ def option(input)
     # rubocop:enable Metrics
   end
 
+  def exit_app
+    save_music_album
+    save_genre
+    puts
+    puts 'Thank You for using my myCatalog Library!'
+    puts 'Built with 💖 by Bright | Dejan | Alick 🗽'
+  end
+
```

Comment:
List of Genre: 
0: Genre: Name: Break
1: Genre: Name: Break
List of all genres is giving same genre twice with different indexes, please fix it

Reply:
Done!